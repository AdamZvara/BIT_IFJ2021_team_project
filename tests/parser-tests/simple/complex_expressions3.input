require "ifj21"


function foo()
	local a : string = "abc".."efd"
	local n1 : number = 2.1E-12
	local n2 : number = 002E1
	local res : number = n1 * n2
	write(res)
	write("\n")
	if ((5*2)-12) ~= (((2+2)*1)+11) then
		n1 = 2.5
		write(n1)
	else end

end

foo()
