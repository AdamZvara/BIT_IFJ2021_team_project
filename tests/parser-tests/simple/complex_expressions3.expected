0x1.716f9798c4336p-35
0x1.4p+1